app.js

```diff
@@ -27,5 +29,39 @@ app.get("/register", (req, res) => {
 app.get("/login", (req, res) => {
   res.render("login");
 });
+app.put("/:type", async(req, res, next) => {
+  if(req.params.type === "student")
+  {
+    const token = await jwtVerify(req.signedCookies.user);
+    const type = token.type;
```

Comment:
What if verification fails? This causes `token.type` to be invalid. You should only search the token if the verification passes.

Reply:
9a4a464832b8a05a2ec4f1893f42397a3c4a522b